tests/components/mqtt/test_light.py

```diff
@@ -650,10 +650,27 @@ async def test_brightness_from_rgb_controlling_scale(
     state = hass.states.get("light.test")
     assert state.attributes.get("brightness") == 255
 
-    async_fire_mqtt_message(hass, "test_scale_rgb/rgb/status", "127,0,0")
+    async_fire_mqtt_message(hass, "test_scale_rgb/rgb/status", "128,64,32")
+
+    state = hass.states.get("light.test")
+    assert state.attributes.get("brightness") == 128
```

Comment:
We should not just check the brightness, we should also check the rgb_color attribute